chromium_src/ui/color/ui_color_mixer.cc

```diff
@@ -52,8 +52,7 @@ void AddBraveUiColorMixer(ColorProvider* provider,
   // Link colors
   // --------------------------------------------------------------------------
   // For deprecated kColorId_LinkEnabled & kColorId_LinkPressed
-  mixer[kColorLinkForeground] = {dark_mode ? gfx::kBraveColorOrange300
-                                           : gfx::kBraveColorBrand};
+  mixer[kColorLinkForeground] = {gfx::kBraveColorBrand};
```

Comment:
Need link color for dark mode.

Reply:
cc: @aguscruiz

Reply:
I think it's probably `--leo-color-text-interactive` which is `rgb(124, 145, 255)` in dark mode

Reply:
optional: can you use colors from Nala?

`leo::GetColor(leo::Color::kColorTextInteractive, leo::Theme::kLight/kDark)`

Reply:
looks good
<img width="590" alt="image" src="https://github.com/brave/brave-core/assets/6786187/b4bd3bfc-8943-43df-be11-452cf51cee39">